src/deephaven/plugin/matplotlib/__init__.py

```diff
@@ -157,5 +157,5 @@ def _step(self, update, *args):
     def _draw_frame(self, framedata):
         data = {}
         for column in self._columns:
-            data[column] = dhnp.to_numpy(self._table, [column])
+            data[column] = dhnp.to_numpy(self._table, [column])[:, 0]
```

Comment:
It might be slightly better for performance if we change to 
```
np_2d = dhnp.to_numpy(self._table, self._columns)
for i, column in enumerate(self._columns):
    data[column]=np_2d[:, i]
```

Reply:
That doesn't work if columns are different types:
```
r-Scheduler-Serial-1 | .c.ConsoleServiceGrpcImpl | Error running script: java.lang.RuntimeException: Error in Python interpreter:
Type: <class 'deephaven.dherror.DHError'>
Value: columns must be of the same data type. : columns must be of the same data type.
NoneType: None
```
I thought about grouping by column type, but I didn't think it would make much of a difference at that point.

Reply:
Agreed.